platforms/documentation/docs/src/docs/userguide/best-practices/bp_general.adoc

```diff
@@ -178,3 +178,38 @@ include::sample[dir="snippets/bestPractices/avoidInternal/groovy",files="build.g
 === Tags
 
 `<<tags_reference.adoc#tag:upgrades,#upgrades>>`
+
+[[avoid_duplicate_dependencies]]
```

Comment:
❗ I don't think we want to change this file, the `bp_dependencies.adoc` change is all that is needed.

Reply:
yeah, this is mistake I overlooked. 
thank you

Reply:
fixed